core/src/main/resources/lib/hudson/scriptConsole.jelly

```diff
@@ -53,7 +53,11 @@ THE SOFTWARE.
           <st:adjunct includes="org.kohsuke.stapler.codemirror.mode.groovy.groovy"/>
           <st:adjunct includes="org.kohsuke.stapler.codemirror.theme.default"/>
           <j:if test="${output!=null}">
-            <h2>${%Result}</h2>
+            <h2>
+              ${%Result}
+              <st:nbsp/>
```

Comment:
can you use css rather than a nbsp space?

Reply:
The nbsp is actually obsolete, there's already a space introduced.